macros/incremental_utils.sql

```diff
@@ -6,3 +6,11 @@
       true
     {% endif %}
 {%- endmacro %}
+
+{% macro incremental_last_x_days(time_col, time_in_days) -%}
+    {% if is_incremental() %}
+        {{ time_col }} >= current_date() - {{ time_in_days }}
+    {% else %}
+        true
+    {% endif %}  
+{%- endmacro %}
```

Comment:
is this in another PR as well? probably just merge the first one and then rebase?